Packs/EWS/Integrations/EWSv2/EWSv2.py

```diff
@@ -105,6 +105,10 @@
 SERVER_BUILD = ""
 MARK_AS_READ = demisto.params().get('markAsRead', False)
 MAX_FETCH = min(50, int(demisto.params().get('maxFetch', 50)))
+FETCH_TIME = demisto.params().get('fetch_time', '10 minutes')
```

Comment:
Assume you may also get here an empty string as the param is optional. I would do the following:
```suggestion
FETCH_TIME = demisto.params().get('fetch_time') or '10 minutes'
```